src/runtime/cmd/kata-runtime/kata-check_s390x.go

```diff
@@ -55,7 +55,14 @@ func kvmIsUsable() error {
 }
 
 func archHostCanCreateVMContainer(hypervisorType vc.HypervisorType) error {
-	return kvmIsUsable()
+	switch hypervisorType {
+	case "qemu":
+		return kvmIsUsable()
+	case "remote":
+		return nil
+	default:
```

Comment:
Aren't Firecracker and Cloud Hypervisor supported on s390x?

Reply:
They will possibly be supported on s390x. I'll fix the logic.

Reply:
done